tensorflow/python/ops/array_ops_test.py

```diff
@@ -143,6 +143,32 @@ def testLowerBoundInputsWrongRank(self):
     ):
       gen_array_ops.lower_bound(arg0, arg1)
 
+  def testWrapPadOp(self):
+    # Op implemented in Python so test is found here.
+    t = array_ops.constant([[1, 2, 3], [4, 5, 6]])
+    paddings = array_ops.constant([[1, 1,], [2, 2]])
+    self.assertAllEqual(
+      array_ops._wrap_pad(t, paddings), array_ops.constant([
+                                      [5, 6, 4, 5, 6, 4, 5],
+                                      [2, 3, 1, 2, 3, 1, 2],
+                                      [5, 6, 4, 5, 6, 4, 5],
+                                      [2, 3, 1, 2, 3, 1, 2]] )
+    )
+
+  def testWrapPadConcreteOp(self):
+    wrap_pad_concrete = array_ops._wrap_pad.get_concrete_function(
+      tensor_spec.TensorSpec(shape=[2, 3], dtype=dtypes.int32), 
+      tensor_spec.TensorSpec(shape=[2, 2], dtype=dtypes.int32))
```

Comment:
shape=None here to test for polymorphism

Reply:
Done